inc/Calculator.h

```diff
@@ -11,6 +11,8 @@
 #define CALCULATOR_H
```

Comment:
1- Need to use #if #def
2- The top of the file is missing the general header. Look at the Dune coding standards for an example